docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -1,4 +1,128 @@
-# Deploy to Fly.io with SQLite
+# Deploy Platformatic Applications to Fly.io
+
+## Deploying a Platformatic Runtime Application 
+
+This guide provides instructions on deploying a Platformatic Runtime application to Fly.io. With a runtime application, you are deploying your entire application, including all services in the `services` folder.
+
+
+### Dockerfile for Runtime Application
+
+Here is an example Dockerfile for a Platformatic Runtime application:
+
+```dockerfile
+FROM node:20-alpine
+
+ARG VITE_AI_URL
+ENV VITE_AI_URL=$VITE_AI_URL
```

Comment:
Why do we need VITE here?

Reply:
The frontend build is done by Vite, It's also a Vite (React) application

Reply:
I don't see any other mention of vite here

Reply:
Made a commit to remove it